test/e2e/snaps/test-snap-bip-32.spec.js

```diff
@@ -28,14 +27,19 @@ describe('Test Snap bip-32', function () {
           tag: 'h2',
         });
 
-        // find and scroll to the bip32 test and connect
+        // find and scroll to the bip32 snap
         const snapButton1 = await driver.findElement('#connectbip32');
         await driver.scrollToElement(snapButton1);
+
+        // added delay for firefox (deflake)
+        await driver.delay(1000);
```

Comment:
Wondering if it's possible to conditionally add the delay for just firefox?

Reply:
Yes it is, and now is. 
Added a function in `driver.js` called `delayFirefox` which only delays if firefox is the current browser.